sdk/aqueduct/error.py

```diff
@@ -130,3 +130,8 @@ class InvalidGithubQueryError(Error):
 # Exception raised when client fails to validate the server.
 class ClientValidationError(Error):
     pass
+
+
+# Exception raised when requirements.txt file is missing
+class RequirementsFileMissingError(Error):
```

Comment:
Sorry it's a super small language nit but I wanted this to just be `RequirementsMissingError` without the "file" part, since requirements can be specified in multiple ways.